.travis.yml

```diff
@@ -0,0 +1,76 @@
+arch: arm64-graviton2
+dist: focal
+virt: vm
+group: edge
+language: python
+services: docker
+env:
+  global:
+      - REPO_DIR=pandas
+      # on release change this to a version
+      - BUILD_COMMIT="v1.1.2"
+      - UNICODE_WIDTH=32
+      - PLAT=aarch64
+      - NP_BUILD_DEP="numpy==1.13.3"
+      - NP_TEST_DEP="numpy==1.13.3"
+      - CYTHON_BUILD_DEP="cython==0.29.16"
+      - WHEELHOUSE_UPLOADER_USERNAME=travis-worker
+      - TEST_DEPENDS="pytest pytest-xdist hypothesis"
+      - TEST_DIR="tmp_for_test"
+      - MB_ML_VER=2014
+      - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+      # Following generated with
+      # travis encrypt -r MacPython/pandas-wheels WHEELHOUSE_UPLOADER_SECRET=<the api key>
+      - secure:
+          "FUNzoxHJjaTGlki4FTwW+/JsVfyulQHIAVzoc126SOWXn3noeEtW6xMurYGFSASYBT+dcPafo6HUBMlxG6a8BWjOnT5mjTWQWRYcVG435rSxrbtbZ9RvkL+E+uquDemjCEeesmCD397kYj96ZoQPbLy/B11IlaTAremyoesmA7g="
+
+jobs:
+  include:
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP="numpy==1.13.3"
+        - NP_TEST_DEP="numpy==1.13.3"
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.7
+        - NP_BUILD_DEP="numpy==1.14.5"
+        - NP_TEST_DEP="numpy==1.14.5"
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.8
+        - NP_BUILD_DEP="numpy==1.19.1"
+        - NP_TEST_DEP="numpy==1.19.1"
+before_install:
+    # See:
+    # https://github.com/travis-ci/travis-ci/issues/8920#issuecomment-352661024
+    - python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+    - CONTAINER="wheels";  # pre-relesae
```

Comment:
Why is this marked pre-release?

Reply:
I based this off of .travis.yml.back. I'll remove the 'pre-release'.